Products/Poi/events.py

```diff
@@ -86,7 +85,11 @@ def remember_tracker(new_issue, event):
 
 
 def assign_id(new_issue, event):
-    """Auto-increment ID numbers"""
+    """Auto-increment ID numbers when they are created.
+       Don't run on copied issues
+    """
+    if new_issue.id.find('copy') >= 0:
+        return
```

Comment:
I opened a new ticket when I committed this, because even in the same tracker it gets an id like `copy_of_7`. Plone will prevent it from getting a conflicting ID in another tracker, but it still wouldn't be an integer. This commit at least allows issues to be successfully copied and pasted.